### DISCUSSION TITLE
How do you determine whether the consumers of a service are using application-level discovery from Dubbo Admin?

### DISCUSSION THREAD

**Discussion by iwangjie** [apache/dubbo#13893] [General]
We have switched to dual registration for all services, but are not sure if the application-level discovery is fully valid. Can We provide a verification idea? Thanks!

**Comment by AlbumenJ** [comment]
> We have switched to dual registration for all services, but are not sure if the application-level discovery is fully valid. Can We provide a verification idea? Thanks!

It seems that dubbo admin will show this service is register to application-level or interface-level in the service detail page.

**Comment by iwangjie** [comment]
> > We have switched to dual registration for all services, but are not sure if the application-level discovery is fully valid. Can We provide a verification idea? Thanks!
> 
> It seems that dubbo admin will show this service is register to application-level or interface-level in the service detail page.

Yes. That's true. I'm pretty sure that the application-level registration of the service provider is in effect, but I'm not sure that consumers really only discover providers at the consumer application level. I can't see at the dubbo admin.

However, if the provider's service name is not consistent with the consumer's `providerBy` fill-in, it is not easy to find, and the provider may not be found after the switch.
